Tests/SwiftLintFrameworkTests/TestHelpers.swift

```diff
@@ -294,11 +294,17 @@ private func testCorrection(_ correction: (Example, Example),
 }
 
 private func addEmoji(_ example: Example) -> Example {
-    return example.with(code: "/* 👨‍👩‍👧‍👦👨‍👩‍👧‍👦👨‍👩‍👧‍👦 */\n\(example.code)")
+    if example.testMultiByteOffsets {
+        return example.with(code: "/* 👨‍👩‍👧‍👦👨‍👩‍👧‍👦👨‍👩‍👧‍👦 */\n\(example.code)")
+    }
+    return example
```

Comment:
Instead of returning the passed example here, why not avoid calling this function altogether if the example disables testing with multi-byte offsets?

Right now, doesn't this cause the example to be verified multiple times?

Reply:
Right. The example was run twice.